test/classes/delete-free/multiple-modifiers.good

```diff
@@ -0,0 +1,38 @@
+multiple-modifiers.chpl:5: error: Type expression uses multiple class kinds: borrowed borrowed
+multiple-modifiers.chpl:6: error: Type expression uses multiple class kinds: borrowed owned
+multiple-modifiers.chpl:7: error: Type expression uses multiple class kinds: borrowed unmanaged
+multiple-modifiers.chpl:8: error: Type expression uses multiple class kinds: borrowed shared
+multiple-modifiers.chpl:9: error: Type expression uses multiple class kinds: owned borrowed
+multiple-modifiers.chpl:10: error: Type expression uses multiple class kinds: owned owned
+multiple-modifiers.chpl:11: error: Type expression uses multiple class kinds: owned unmanaged
+multiple-modifiers.chpl:12: error: Type expression uses multiple class kinds: owned shared
+multiple-modifiers.chpl:13: error: Type expression uses multiple class kinds: unmanaged borrowed
+multiple-modifiers.chpl:14: error: Type expression uses multiple class kinds: unmanaged owned
+multiple-modifiers.chpl:15: error: Type expression uses multiple class kinds: unmanaged unmanaged
+multiple-modifiers.chpl:16: error: Type expression uses multiple class kinds: unmanaged shared
+multiple-modifiers.chpl:17: error: Type expression uses multiple class kinds: shared borrowed
+multiple-modifiers.chpl:18: error: Type expression uses multiple class kinds: shared owned
+multiple-modifiers.chpl:19: error: Type expression uses multiple class kinds: shared unmanaged
+multiple-modifiers.chpl:20: error: Type expression uses multiple class kinds: shared shared
+multiple-modifiers.chpl:23: error: Type expression uses multiple class kinds: unmanaged shared
+multiple-modifiers.chpl:22: error: Type expression uses multiple class kinds: owned unmanaged
+multiple-modifiers.chpl:21: error: Type expression uses multiple class kinds: borrowed owned
```

Comment:
Don't ask me how I caught this but these numbers are out of order - I suppose it is happening because it is telling you all the multiple combinations in a single statement (yay!) and going in reverse order from there.  I don't think this is worth fixing, but figured I should point it out anyways

Reply:
I know they're out of order, it has to do with the order of things in gCallExprs, but it would be out of order in other situations too.